2018-02-24 19:16:37,889 ERROR 1772 [-/::1/-/1ms POST /api/auth/logout] nodejs.ReferenceError: app is not defined
    at AuthController.logout (E:\jtStdio\eggT\demo1\egg-example\app\controller\auth.js:84:20)
    at Object.callFn (E:\jtStdio\eggT\demo1\egg-example\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (E:\jtStdio\eggT\demo1\egg-example\node_modules\egg-core\lib\loader\mixin\controller.js:84:20)
    at Object.callFn (E:\jtStdio\eggT\demo1\egg-example\node_modules\egg-core\lib\utils\index.js:44:21)
    at wrappedController (E:\jtStdio\eggT\demo1\egg-example\node_modules\egg-core\lib\utils\router.js:322:18)
    at dispatch (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-router\lib\router.js:346:16
    at dispatch (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-compose\index.js:42:32)
    at next (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-compose\index.js:43:18)
    at saveSession (E:\jtStdio\eggT\demo1\egg-example\app\middleware\save_session.js:5:11)
    at dispatch (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-compose\index.js:42:32)
    at next (E:\jtStdio\eggT\demo1\egg-example\node_modules\koa-compose\index.js:43:18)

pid: 1772
hostname: SF0001367391A

2018-02-24 19:38:58,794 ERROR 10532 [-/::1/-/54ms POST /api/auth/login] nodejs.TypeError: app.redis.get(...).set is not a function
    at AuthController.login (E:\jtStdio\eggT\demo1\egg-example\app\controller\auth.js:55:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

pid: 10532
hostname: SF0001367391A

2018-02-24 19:41:00,391 ERROR 11152 [-/::1/-/56ms POST /api/auth/login] nodejs.ReplyError: ERR wrong number of arguments for 'set' command
    at JavascriptReplyParser.returnError (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\redis\parser.js:25:25)
    at JavascriptReplyParser.run (E:\jtStdio\eggT\demo1\egg-example\node_modules\redis-parser\lib\javascript.js:135:18)
    at JavascriptReplyParser.execute (E:\jtStdio\eggT\demo1\egg-example\node_modules\redis-parser\lib\javascript.js:112:10)
    at Socket.<anonymous> (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\redis\event_handler.js:107:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:594:20)
From previous event:
    at Command.initPromise (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\command.js:63:18)
    at new Command (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\command.js:51:8)
    at Redis.set (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\commander.js:131:29)
    at AuthController.login (E:\jtStdio\eggT\demo1\egg-example\app\controller\auth.js:55:23)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
name: 'ReplyError'
message: 'ERR wrong number of arguments for \'set\' command'
command: { name: 'set', args:  [ 'instance1', 'userInfo', '{"id":309,"jobcard":"01367391","username":"jt"}' ] }
pid: 11152
hostname: SF0001367391A

2018-02-24 19:47:53,152 ERROR 8028 [-/::1/-/57ms POST /api/auth/login] nodejs.TypeError: app.redis.set(...).set is not a function
    at AuthController.login (E:\jtStdio\eggT\demo1\egg-example\app\controller\auth.js:55:40)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

pid: 8028
hostname: SF0001367391A

2018-02-24 19:47:53,159 ERROR 8028 nodejs.ReplyError: ERR wrong number of arguments for 'set' command
    at JavascriptReplyParser.returnError (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\redis\parser.js:25:25)
    at JavascriptReplyParser.run (E:\jtStdio\eggT\demo1\egg-example\node_modules\redis-parser\lib\javascript.js:135:18)
    at JavascriptReplyParser.execute (E:\jtStdio\eggT\demo1\egg-example\node_modules\redis-parser\lib\javascript.js:112:10)
    at Socket.<anonymous> (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\redis\event_handler.js:107:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:594:20)
From previous event:
    at Command.initPromise (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\command.js:63:18)
    at new Command (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\command.js:51:8)
    at Redis.set (E:\jtStdio\eggT\demo1\egg-example\node_modules\ioredis\lib\commander.js:131:29)
    at AuthController.login (E:\jtStdio\eggT\demo1\egg-example\app\controller\auth.js:55:23)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
name: 'ReplyError'
message: 'ERR wrong number of arguments for \'set\' command'
command: { name: 'set', args: [ 'instance1' ] }
pid: 8028
hostname: SF0001367391A

2018-02-24 19:48:46,984 ERROR 12280 [-/::1/-/56ms POST /api/auth/login] nodejs.TypeError: Cannot read property 'userInfo' of null
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
From previous event:
    at AuthController.login (E:\jtStdio\eggT\demo1\egg-example\app\controller\auth.js:60:55)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

pid: 12280
hostname: SF0001367391A

2018-02-24 19:49:18,837 ERROR 5528 [-/::1/-/56ms POST /api/auth/login] nodejs.TypeError: Cannot read property 'userInfo' of null
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
From previous event:
    at AuthController.login (E:\jtStdio\eggT\demo1\egg-example\app\controller\auth.js:60:55)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

pid: 5528
hostname: SF0001367391A

2018-02-24 21:05:09,633 ERROR 11124 nodejs.ECONNRESETError: read ECONNRESET
    at _errnoException (util.js:1024:11)
    at TCP.onread (net.js:615:25)
code: 'ECONNRESET'
errno: 'ECONNRESET'
syscall: 'read'
name: 'ECONNRESETError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:09,647 ERROR 11124 nodejs.ECONNRESETError: read ECONNRESET
    at _errnoException (util.js:1024:11)
    at TCP.onread (net.js:615:25)
code: 'ECONNRESET'
errno: 'ECONNRESET'
syscall: 'read'
name: 'ECONNRESETError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:10,650 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:10,659 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:11,659 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:11,668 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:12,668 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:12,675 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:13,685 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:13,690 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:14,701 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:14,703 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:15,715 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:15,718 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:16,732 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:16,736 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:17,753 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:17,758 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:18,776 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:18,783 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:19,797 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:19,804 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:20,821 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:20,827 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:21,841 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:21,852 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:22,884 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:22,890 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:23,930 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:23,931 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:24,961 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:24,968 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:26,004 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:26,009 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:27,043 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:27,051 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:28,083 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:28,091 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:29,127 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:29,136 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:30,175 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:30,180 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:31,223 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:31,227 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:32,271 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:32,277 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:33,323 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:33,333 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:34,376 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:34,387 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:35,431 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:35,441 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:36,491 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:36,497 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:37,549 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:37,556 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:38,609 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:38,618 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:39,681 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:39,691 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:40,742 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:40,752 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:41,809 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:41,831 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:42,875 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:42,899 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:43,949 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:43,971 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:45,031 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:45,049 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:46,105 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:46,125 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:47,185 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:47,205 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:48,265 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:48,288 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:49,345 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:49,365 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:50,429 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:50,448 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:51,520 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:51,528 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:52,605 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:52,617 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:53,700 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:53,711 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:54,793 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:54,804 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:55,882 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:55,897 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:56,986 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:56,993 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:58,085 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:58,096 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:59,185 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:05:59,195 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:06:00,286 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:06:00,298 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:06:01,396 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

2018-02-24 21:06:01,403 ERROR 11124 nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:6379
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
code: 'ECONNREFUSED'
errno: 'ECONNREFUSED'
syscall: 'connect'
address: '127.0.0.1'
port: 6379
name: 'ECONNREFUSEDError'
pid: 11124
hostname: SF0001367391A

